### PR TITLE
Add some docs on why we created strawberry and why use it

### PR DESCRIPTION
**Pull request by patrick91**

### CODE REVIEW

File: docs/01_general/05_why.md
Comment (BryceBeagle) [inline docs/01_general/05_why.md:13]:
```suggestion
community with people using GraphQL in Python.
```

File: docs/01_general/05_why.md
Comment (BryceBeagle) [inline docs/01_general/05_why.md:18]:
```suggestion
> chance of things suddenly changing, but hopefully the public API is stable
```

File: docs/01_general/05_why.md
Comment (BryceBeagle) [inline docs/01_general/05_why.md:25]:
```suggestion
We also are going to provide useful features and integrations; for example we
```

File: docs/01_general/05_why.md
Comment (BryceBeagle) [inline docs/01_general/05_why.md:29]:
```suggestion
Finally, we try to fix bugs and provide help via GitHub or our Discord server,
```

We might want to have a link the discord server here, as well?

File: docs/01_general/05_why.md
Comment (BryceBeagle) [inline docs/01_general/05_why.md:30]:
```suggestion
so if you have any issue feel free to drop an issue or ask on Discord.
```

**Review by BryceBeagle — CHANGES_REQUESTED** [review]

File: docs/01_general/05_why.md
Comment (patrick91) [reply docs/01_general/05_why.md:29]: good idea, I'll see if I can make a `discord.strawberry.rocks` link tomorrow :)

**Comment by Ambro17** [comment]
The thing I like the most about strawberry compared to graphene is its clean syntax when defining gql types.
I don't know if it's the correct place in the docs, but a verbosity comparison between graphene and strawberry would be really cool.

I would also mention that this clean syntax helps maintaining focus on your api entities and the generated SDL.
It's common to get lost on the classes with graphene and lose visibility on the generated SDL.

**Comment by patrick91** [comment]
> The thing I like the most about strawberry compared to graphene is its clean syntax when defining gql types.
> I don't know if it's the correct place in the docs, but a verbosity comparison between graphene and strawberry would be really cool.

You think? I'm worried that it would sound like bragging or just being bad towards Graphene :)


> I would also mention that this clean syntax helps maintaining focus on your api entities and the generated SDL.
> It's common to get lost on the classes with graphene and lose visibility on the generated SDL.
I have some ideas on how to make that better, I'd love to have a tool (maybe a vscode plugin) that allows you to jump from a query to the the implementation.

> 
> Also, I think strawberry hits a perfect middle ground between code first and schema first.
> As code is really close (in my opinion) to the generated schema.
That's true, type hints and classes looks similar to the SDL, not sure how I'd explain that though :)

> So it's easier to follow a design first approach with strawberry than any other python graphql lib I tried

**Comment by Ambro17** [comment]
> > The thing I like the most about strawberry compared to graphene is its clean syntax when defining gql types.
> > I don't know if it's the correct place in the docs, but a verbosity comparison between graphene and strawberry would be really cool.
> 
> You think? I'm worried that it would sound like bragging or just being bad towards Graphene :)


We could avoid naming the lib explicitly 😜
To be honest, I think most users of graphene will eventually look for alternatives, and my idea of this doc section was giving an answer to them, explaining the design choices of strawberry so they can decide if the things that they dislike about graphene are tackled differently in strawberry. 

AFAIK you pay special attention to the usability and intuitiveness of the API, and that is reflected in the lib by the general absence of boilerplate to get things done. 

For example, some questions a graphene user may have may be :

Why is there another lib if graphene already has a lot of traction?
In what ways it's different and _why_? 
Is it ready for production? 
What painpoints it intends to mitigate?

I really liked the mention to the community aspect, it's really rare to see a maintainer of a popular lib that is open minded that welcomes suggestions and improvements.

Perhaps we could also share tweets from Devs using strawberry, it's pretty common on landing pages (although they don't always seem organic so idk)

**Comment by BryceBeagle** [comment]
> You think? I'm worried that it would sound like bragging or just being bad towards Graphene :)

I think we could do this in a pretty friendly way. "Graphene is great. If you're used to [this], then you'll love [this] even more".

**Comment by AndrewIngram** [comment]
I think it's possible to avoid loaded phrases like "clean syntax" and try to come up with a value-led explanation as to why you prefer the dataclass syntax over the Django-style declarative metaclass approach. It's reasonable to talk about alternatives by name, as long as it's respectful.

**Comment by botberry** [comment]
Hi, thanks for contributing to Strawberry 🍓!

We noticed that this PR is missing a `RELEASE.md` file. We use that to automatically do releases here on GitHub and, most importantly, to PyPI!

So as soon as this PR is merged, a release will be made 🚀.

Here's an example of `RELEASE.md`:

```markdown
Release type: patch

Description of the changes, ideally with some examples, if adding a new feature.
```

Release type can be one of patch, minor or major. We use [semver](https://semver.org/),so make sure to pick the appropriate type. If in doubt feel free to ask :)

**Comment by Ambro17** [comment]
The docs are looking great!
I would add a comparison with graphene and ariadne, perhaps with the same user entity to compare verbosity and dev experience when adding a resolver (Ariadne api on this is rather strange)

**Comment by patrick91** [comment]
@Ambro17 nice idea! I'll merge this and send another PR later then 😊

**Pull request merged**
